lib/index.js

```diff
@@ -61,14 +61,17 @@ const PERMISSIONS = bedrock.config.permission.permissions;
  *   tokens, `serviceId` is encoded in the `otpAuthUrl` value returned
  *   by this API.
  * @param [notify=true] `true` to notify the account user, `false` not to.
+ * @param [typeOptions={entryStyle: 'human'}] Setting entryStyle to `human`
+ *   will generate 9 character numeric-only nonce and setting it to `machine`
+ *   will generate 128-bit bnid.
```

Comment:
Describe in generic terms since 'bnid' is an implementation detail.
```suggestion
 *   will generate 128-bit base58 encoded nonce.
```